CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# dbt_hubspot_source v0.7.1 (May be a breaking change, TBD)
```

Comment:
imo this is a breaking change since the data they end up with will be different by default (they'll have deleted records they didn't before). should be something they're aware of?

Reply:
You make a good point! I was leaning toward breaking change, this helps cement it. Thank you!